### PR TITLE
code added to get route tables automatically for VPC peering connection

### PR DESCRIPTION
**Pull request by urvi-occ**
Jira Ticket: [DDI-106](https://occ-data.atlassian.net/browse/DDI-106)

- This script take an optional flag "--get-route-table" to find the route table id of the VPC which is accepting the peering request

### New Features
Added new optional flag "--get-route-table"

### Improvements
when the older script was taking manual entries for route table, this will allow an option to programmatically get route table(s) from the accepter VPC

### CODE REVIEW

File: gen3/bin/approve_vpcpeering_request.sh
Comment (jiaqi216) [inline gen3/bin/approve_vpcpeering_request.sh:3]: Would be helpful to update the Usage and/or add comments on the new flag that's being introduced and how to use it

File: gen3/bin/approve_vpcpeering_request.sh
Comment (jiaqi216) [inline gen3/bin/approve_vpcpeering_request.sh:13]: Is this a hard coded route table ID? I guess that's fine since it looks like we've been hardcoding this all along but `rtb-rtb-7ee06301` should probably be `rtb-7ee06301` right?

**Review by jiaqi216 — COMMENTED** [review]
Left some initial comments but will defer to a devops member for final review.

File: gen3/bin/approve_vpcpeering_request.sh
Comment (urvi-occ) [reply gen3/bin/approve_vpcpeering_request.sh:3]: Ya sure. I'll add an readme for it.

File: gen3/bin/approve_vpcpeering_request.sh
Comment (urvi-occ) [reply gen3/bin/approve_vpcpeering_request.sh:13]: Yeah the route ID was hardcoded in the previous script so I left it just in case. I am not sure about route table id `rtb-rtb-7ee06301` I guess it is supposed to be `rtb-7ee06301`. It needs to be checked in CTDS CSOC.

File: CSOC_Documentation.md
Comment (jiaqi216) [inline CSOC_Documentation.md:66]: a couple of typos:

`scrirpt` -> `script`
`programtically` ->`programmatically`

File: gen3/bin/approve_vpcpeering_request.sh
Comment (jiaqi216) [reply gen3/bin/approve_vpcpeering_request.sh:13]: I think if you preserve what was hardcoded in the script before (i.e. https://github.com/uc-cdis/cloud-automation/pull/1534/files#diff-49fb1a0d6091e1ba2697cbea00f1fc6f8a04004d761459bf0184fe1ed1ef30ecL18)  that would be fine going forward as well. With that said I don't think `rtb-rtb-####` is a valid Route Table ID so it would make sense to correct that

Comment (urvi-occ) [reply gen3/bin/approve_vpcpeering_request.sh:13]: I considered the way to preserve hardcoded values while coding and tested it so it can work with the hardcoded value when the flag is not passed. About the route table ID, can you help me with getting value of the route table id ?

Comment (urvi-occ) [reply gen3/bin/approve_vpcpeering_request.sh]: I completely agree. I made sure while changing the code in the file to preserve the hardcoded value. If in case the flag is not passed, the script will use the hardcoded values. I also made change for route table ID as before.

**Review by jawadqur — APPROVED** [review]
Looks good to me! Thanks for fixing the last typos. I will wait for Jenkins to complete before merging this 👍

File: CSOC_Documentation.md
Comment (urvi-occ) [reply CSOC_Documentation.md]: Typos Fixed

**Pull request merged**
